Gulpfile.ts

```diff
@@ -748,7 +745,7 @@ gulp.task("runtests",
 const nodeServerOutFile = "tests/webTestServer.js";
 const nodeServerInFile = "tests/webTestServer.ts";
 gulp.task(nodeServerOutFile, false, [servicesFile], () => {
-    const settings: tsc.Settings = getCompilerSettings({}, /*useBuiltCompiler*/ true);
+    const settings: tsc.Settings = getCompilerSettings({module: "commonjs"}, /*useBuiltCompiler*/ true);

```

Comment:
I'm not sure off the top of my head. I "read somewhere" that the Javascript standard syntax was with extra spaces, but I don't remember how *actually* authoritative the source was. I'll try to track down the source.